runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/fn/BeamFnControlServiceTest.java

```diff
@@ -88,6 +88,8 @@ public void testClientConnecting() throws Exception {
     server.shutdown();
     server.awaitTermination(1, TimeUnit.SECONDS);
     server.shutdownNow();
+    Thread.sleep(1000);  // Wait for stub to close stream.
```

Comment:
Fixing a flake is important enough that we can merge this, but please keep a ticket open. If you have time and inclination to fix the test to use a latch instead of a sleep, that would be preferable. I imagine you can do this with the mocks by inserting code to run when onCompleted is called, or you could move away from mocks since StreamObserver is easy to implement.